advanced/config.yaml

```diff
@@ -0,0 +1,114 @@
+components:
+  schemas:
+    Advanced.Config:
+      description: |-
+        The container for the configuration of advanced features.
+      type: object
+      properties:
+        flows:
+          $ref: '#/components/schemas/Advanced.Config.Flows'
+        timestamps:
+          $ref: '#/components/schemas/Advanced.Config.Timestamp'
+        latency:
+          $ref: '#/components/schemas/Advanced.Config.Latency'
+        event:
+          $ref: '#/components/schemas/Advanced.Config.Event'
+        convergence:
+          $ref: '#/components/schemas/Advanced.Config.Convergence'
+
+    Advanced.Config.Flows:
+      description: |-
+        The optional container for flows metric configuration.
+      type: object
+      properties:
+        enable:
```

Comment:
What happens if somebody has not explicitly enabled advanced.config.flows but has enabled advanced.config.latency or advanced.config.timestamps?  Should that request get rejected? Or is it implementation dependent?

Reply:
That will be implementation independent